website/blog/2020-05-25-7.10.0.md

```diff
@@ -0,0 +1,184 @@
+---
+layout: post
+title:  "7.10.0 Released: Class Fields in preset-env, '#private in' checks and better React tree-shaking"
+author: Nicolò Ribaudo
+authorURL: https://twitter.com/NicoloRibaudo
+date:   2020-05-25 0:00:00
+categories: announcements
+share_text: "Babel 7.10.0 Released"
+---
+
+We just released a new minor version of Babel!
+
+This 7.10 release includes:
+- Full support for the new Stage 1 proposal, `#prop in obj` checks for private fields [proposal](https://github.com/tc39/proposal-private-fields-in-in) (`class A { #a = 1 }`).
```

Comment:
```suggestion
- Full support for the new Stage 1 proposal, `#prop in obj` checks for private fields [proposal](https://github.com/tc39/proposal-private-fields-in-in) (`class A { #prop; static isA(obj) { return #prop in obj } }`).
```

The code example should include brand checks, though it seems more verbose.

Reply:
Maybe just `#prop in obj` to keep it short?

Reply:
```suggestion
- Full support for the new Stage 1 proposal, `#prop in obj` checks for private fields [proposal](https://github.com/tc39/proposal-private-fields-in-in) (`#prop in obj`).
```

Reply:
Or we can just delete code example since we have mentioned it before.